Assignment/solution_student/index.html

```diff
@@ -4,30 +4,95 @@
     <meta charset="UTF-8">
     <title>Relaxr Blog</title>
     <meta name="description" content="An HTML Starter Template">
-    
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,600italic,400italic,800,800italic' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
 
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
   <body>
+    
+    <header>
 
-    <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
+      <h1>Relaxr</h1>
+<!--  -->
+        <nav>
+          <ul>
+            <li><a href="#" class="nav-links">About</a></li>
+            <li><a href="#" class="nav-links">FAQ</a></li>
+            <li><a href="#" class="nav-links">Team</a></li>
+            <li><a href="#" class="nav-links">Contact Us</a></li>
+            <li><a href="#" class="nav-links">Blog</a></li>
+          </ul>
+        </nav>
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+    </header>
+
+    <main>
+
+    <div class="main clearfix">
+    <div class="main-content">
```

Comment:
This is totally fine, and you used the container as you should for styling in your CSS, but since you already have a container for this area (the `<main>` element), you could just keep it at that, not needing this second container — I see you're applying `float: left;width:90%;` to it in your CSS, which is just achieving the space on the right side of the container, but I think it'd make more sense to just apply a `padding-right: 10%;` to the `main` element instead, since it is really padding that you're trying to achieve, and might as well avoid using more floats if you can since we all know how much trouble they cause! haha